Lib/annotationlib.py

```diff
@@ -154,8 +155,19 @@ def evaluate(self, *, globals=None, locals=None, type_params=None, owner=None):
                     globals[param_name] = param
                     locals.pop(param_name, None)
 
-        code = self.__forward_code__
-        value = eval(code, globals=globals, locals=locals)
+        arg = self.__forward_arg__
+        if arg.isidentifier():
```

Comment:
Do you also need to check `not keyword.iskeyword(arg)`? (whether yes or no, I think you should add some tests with keywords :-)